src/org/ods/services/OpenShiftService.groovy

```diff
@@ -487,9 +487,10 @@ class OpenShiftService {
         def urlParts = url.split('/').toList()
 
         if (urlParts.size() < 2) {
-            throw new RuntimeException(
-                "ERROR: Image URL ${url} must have at least two parts (repository/reference)"
-            )
+            logger.debug "Image URL ${url} does not define the repository explicitly."
```

Comment:
Was the previous error message not more precise in case the URL does contain the repository but not the reference (for whatever reason)? Or is that not possible for some reason?

Reply:
Previously calling this method required the image to look like `<repository>/<name>` (`name` is called `reference` in the error message). If the image did not have two parts, an exception was thrown. Now we also allow `<name>` - in that case the debug message is printed, telling us that the repository is not defined.

Reply:
And yes, there always has to be a name ...